From e7005db5b9e3b8728f5cde6c32a3bfad70f51452 Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Wed, 10 Aug 2022 15:21:34 +0300
Subject: [PATCH 27/42] fdts: s32: remove old dt-bindings includes

Those dt-bindings includes were deleted by a previous commit. Remove any
references to them and include the new generic dt-bindings includes.

Issue: ALB-8868

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32g.dtsi        | 2 +-
 fdts/s32g274a-emu.dts | 2 +-
 fdts/s32r45-evb.dts   | 2 +-
 fdts/s32r45.dtsi      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index 909aa381b..3352e6b09 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -5,7 +5,7 @@
 #include <dt-bindings/clock/s32g-clock.h>
 #include <dt-bindings/clock/s32g-scmi-clock.h>
 #include <dt-bindings/nvmem/s32cc-siul2-nvmem.h>
-#include <dt-bindings/pinctrl/s32g-pinctrl.h>
+#include <dt-bindings/pinctrl/s32cc-pinfunc.h>
 #include <dt-bindings/memory/s32-siul2.h>
 
 #include "s32cc.dtsi"
diff --git a/fdts/s32g274a-emu.dts b/fdts/s32g274a-emu.dts
index 5f44035a6..c14616c4b 100644
--- a/fdts/s32g274a-emu.dts
+++ b/fdts/s32g274a-emu.dts
@@ -4,7 +4,7 @@
  */
 
 /dts-v1/;
-#include <dt-bindings/pinctrl/s32g-pinctrl.h>
+#include <dt-bindings/pinctrl/s32cc-pinfunc.h>
 #include "s32g2.dtsi"
 
 / {
diff --git a/fdts/s32r45-evb.dts b/fdts/s32r45-evb.dts
index b8cd118ef..4bca29378 100644
--- a/fdts/s32r45-evb.dts
+++ b/fdts/s32r45-evb.dts
@@ -5,7 +5,7 @@
 #include <dt-bindings/clock/s32r45-scmi-clock.h>
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/nvmem/s32cc-siul2-nvmem.h>
-#include <dt-bindings/pinctrl/s32r45-pinctrl.h>
+#include <dt-bindings/pinctrl/s32cc-pinfunc.h>
 
 /dts-v1/;
 #include "s32r45.dtsi"
diff --git a/fdts/s32r45.dtsi b/fdts/s32r45.dtsi
index 985f02c37..64f162d00 100644
--- a/fdts/s32r45.dtsi
+++ b/fdts/s32r45.dtsi
@@ -7,7 +7,7 @@
 #include <dt-bindings/ddr-errata/s32-ddr-errata.h>
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/nvmem/s32cc-siul2-nvmem.h>
-#include <dt-bindings/pinctrl/s32r45-pinctrl.h>
+#include <dt-bindings/pinctrl/s32cc-pinfunc.h>
 #include <dt-bindings/memory/s32-siul2.h>
 
 #include "s32cc.dtsi"
-- 
2.17.1

